libs/cli/src/generators/theme/libs/add-theme-to-application-styles.ts

```diff
@@ -74,7 +74,7 @@ function findStylesEntryPoint(tree: Tree, project: ProjectConfiguration): string
 	}
 
 	// then check for the specified styles in the build configuration if it exists
-	const styles: Array<string | { input: string; inject: boolean }> = project.targets?.build.options?.styles;
+	const styles: Array<string | { input: string; inject: boolean }> = project.targets?.['build'].options?.styles;
```

Comment:
```suggestion
	const styles: Array<string | { input: string; inject: boolean }> = project.targets?.['build']?.options?.styles;
```
Might not be needed but "seems" logical to keep the chaining all the way through